tests/e2e_test.go

```diff
@@ -45,8 +63,15 @@ func TestE2E(t *testing.T) {
 
 // Define the Relayer before and after suite functions.
 var _ = ginkgo.BeforeSuite(func() {
+	var ctx context.Context
+	ctx, cancelDecider = context.WithCancel(context.Background())
```

Comment:
Maybe rename this to just `cancel` now that it can be used to cancel other things too (eg the build process)